lib/sufia/migration/survey/surveyor.rb

```diff
@@ -0,0 +1,19 @@
+# Class that will survey Fedora based on a list of ids
+module Sufia
+  module Migration
+    module Survey
+      class Surveyor
+        class << self
+          # call causes the surveyor to create a survey item for each id in the list
+          #
+          # @param [Array] id_list a list of ids to be surveyed
+          def call(id_list)
+            ActiveFedora::Base.find(id_list).each do |object|
+              Item.create(object_id: object.id, object_class: object.class, object_title: object.title, migration_status: -1)
```

Comment:
It seems that meaning has been encoded in `migration_status`, but it's not clear to me what `-1` represents.